### DISCUSSION TITLE
Welcome to fingerprint-android Discussions!

### DISCUSSION THREAD

**Discussion by Valve** [fingerprintjs/fingerprintjs-android#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TolaniJospeh** [comment]
Fingerprint is no more working on my phone TECNO spark 9T.
Even the fingerprint icon is not appearing in my settings at all. Please what can I do resolve this

**Reply by Alexey-Verkhovsky** [reply]
Hello @TolaniJospeh. Unfortunately we cannot help you. This is a repository dedicated to a tool for developers and it has nothing to do with the fingerprint sensor problem you described. Please contact support of your phone manufacturer for recommendations. For TECNO you may find it here: https://www.carlcare.com/global/contact/
Hope this helps.
